src/fund_my_watcard/transaction.py

```diff
@@ -0,0 +1,15 @@
+import prettytable
+import re
+
+
+def print_transaction(logfile):
+    table = prettytable.PrettyTable(["Date", "Account", "Fund", "Status"])
+    with open(logfile, "r") as f:
```

Comment:
I try to improve something in this function, what do you think?

Reply:
Also, do you think `print_transactions` is better than `print_transaction`? Since there may more than one transaction. Please also consider whether the name of this python file should be changed.

Reply:
What kind of improvement?

Reply:
> What kind of improvement?

https://github.com/xingweitian/fund-my-watcard/pull/39/commits/29497b56ea60bc64335255128567a35fe225b653#diff-5c90e45cae7dfce7770d3b7731ee3b70R5

Reply:
Cool.